gulpfile.js

```diff
@@ -112,7 +112,7 @@ var buildIndex = function(done) {
 
   if (process.env.CI) {
     writeFileSync('build/index.html', outputIndex);
-    copyRecursiveSync('public/assets', 'build/assets');
+    copyRecursiveSync('assets', 'build/assets');
```

Comment:
do we still need this?

Reply:
Yes I've got a little suggestion from vite at a time telling it will be using files from "assets" instead of public/assets so i reverted back to the original folder.